src/components/Dropdown/OptionsList.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import Option from './Option';
+import ClassList, { initialClass } from '../../utils/ClassList';
+
+const OptionsList = ({
+  dropdownOpen,
+  options,
+  onClick
+}) => {
+  const handleClick = value => () => onClick(value);
+
+  const getClassName = () => {
+    const classes = initialClass(['dropdown__options']);
+    const newClasses = ClassList.of([
+      dropdownOpen
+        ? 'dropdown__options--visible'
```

Comment:
I think It's more manageable having visual things in it's own separate file. It could be improved though